entity-framework/toc.yml

```diff
@@ -45,6 +45,8 @@
       items:
       - name: Current and planned releases
         href: core/what-is-new/index.md
+      - name: NuGet packages
+        href: core/what-is-new/nuget-packages.md
```

Comment:
I'd put this under "Getting started"

Reply:
@JeremyLikness @dotnet/efteam Thoughts about where this page should go?

Reply:
I like "Getting Started."